libs/api-mocks/msw/handlers.ts

```diff
@@ -63,7 +64,7 @@ export const handlers = makeHandlers({
     }
     db.projects.push(newProject)
 
-    return json(newProject, { status: 201 })
+    return HttpResponse.json(newProject, { status: 201 })
```

Comment:
I wouldn’t say no to cutting this diff down by doing `const json = HttpResponse.json`

Reply:
Took a big of finagling but I got it to work. Diff is definitely smaller now.